hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -378,9 +380,13 @@ func (r *HostedClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 }
 
 func (r *HostedClusterReconciler) ReconcileMetadataProvidersImpl(ctx context.Context, imgOverrides map[string]string) (releaseinfo.ProviderWithOpenShiftImageRegistryOverrides, hyperutil.ImageMetadataProvider, error) {
-	releaseProvider, imageMetadataProvider, err := globalconfig.RenconcileMgmtImageRegistryOverrides(ctx, r.ManagementClusterCapabilities, r.Client, imgOverrides)
-
-	return releaseProvider, imageMetadataProvider, err
+	imageRegistryMirrors, err := globalconfig.GetAllImageRegistryMirrors(ctx, r.Client, r.ManagementClusterCapabilities.Has(capabilities.CapabilityIDMS), r.ManagementClusterCapabilities.Has(capabilities.CapabilityICSP))
```

Comment:
can you help me understand how is this different?

Reply:
nvm I thought this was addressing a different thing, but I see is about the every reconcile instantation